packages/twenty-server/src/engine/core-modules/auth/token/services/refresh-token.service.ts

```diff
@@ -103,6 +103,10 @@ export class RefreshTokenService {
       token,
       authProvider: jwtPayload.authProvider,
       targetedTokenType: jwtPayload.targetedTokenType,
+      isImpersonating: jwtPayload.isImpersonating,
+      impersonationType: jwtPayload.impersonationType,
+      impersonatorUserWorkspaceId: jwtPayload.impersonatorUserWorkspaceId,
```

Comment:
I'm confused by the naming
impersonator is the person impersonating
originalUserWorkspace is not clear to me

impersonatorUserWorkspaceId + impersonatedUserWorkspaceId?